metric-anomaly-task-manager/src/main/java/org/hypertrace/alert/engine/metric/anomaly/task/manager/job/AlertTaskConverter.java

```diff
@@ -29,6 +29,7 @@
   static final String DELAY_IN_MINUTES_CONFIG = "delayInMinutes";
   static final String EXECUTION_WINDOW_IN_MINUTES_CONFIG = "executionWindowInMinutes";
   static final String TENANT_ID_CONFIG = "tenant_id";
+  static final String EVENT_CHANNEL_ID = "channelId";
```

Comment:
we don't need this inforamtion in alertTask. Using eventConditionId, the notificationRules can be matched.

Reply:
that will need eventConditionId information to be added in channel details, I think notification channel should not have event related details